packages/larva-patterns/modules/heading/heading.h4.js

```diff
@@ -0,0 +1,7 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const heading = clonedeep( require( './heading.prototype' ) );
+
+heading.heading_level_text = '4';
```

Comment:
Add a property to heading.twig for `heading_typography_class` – we should follow the same approach as customizing other property values e.g. `border_color_class`, etc. Then in the theme configuration, specify which class for the typography styles.